packages/babel-preset/index.js

```diff
@@ -50,8 +52,15 @@ function addDefaultOptions(explicitOptions) {
   );
 
   if (options.useBuiltIns) {
-    // By default, stay with core-js v2.x for now.
-    options.envCorejs = options.corejs || 2;
+    options.envCorejs = options.envCorejs || options.corejs || 3;
```

Comment:
Should this default to `2`? Otherwise this is sort of breaking-ish

Reply:
i figured make it a breaking change. Better to have the default be what we want   I think